examples/events/snowplow-debugger/product.json

```diff
@@ -174,4 +175,4 @@
       "version": "2-0-5"
```

Comment:
I am guessing this is a patch bump to `2-0-6` or would be consider this a minor change to `2-1-0` since this could start rejecting events from stores where the sku field is empty? Also not entirely clear if schema changes are entirely manual and this file is used to reflect what is up in snowplow.